internal/pkg/store/ssm/ssm.go

```diff
@@ -231,10 +215,109 @@ func (s *SSM) ListEnvironments(projectName string) ([]*archer.Environment, error
 	var environments []*archer.Environment
 
 	environmentsPath := fmt.Sprintf(rootEnvParamPath, projectName)
+	serializedEnvs, err := s.listParams(environmentsPath)
+	if err != nil {
+		return nil, err
+	}
+	for _, serializedEnv := range serializedEnvs {
+		var env archer.Environment
+		if err := json.Unmarshal([]byte(*serializedEnv), &env); err != nil {
+			return nil, err
+		}
+
+		environments = append(environments, &env)
+	}
+	return environments, nil
+}
+
+// CreateApplication instanciates a new application within an existing project. Returns ErrApplicationAlreadyExists
+// if the application already exists in the project.
+func (s *SSM) CreateApplication(app *archer.Application) error {
+	if _, err := s.GetProject(app.Project); err != nil {
+		return err
+	}
+
+	applicationPath := fmt.Sprintf(fmtAppParamPath, app.Project, app.Name)
+	data, err := marshal(app)
+	if err != nil {
+		return err
+	}
+
+	paramOutput, err := s.systemManager.PutParameter(&ssm.PutParameterInput{
+		Name:        aws.String(applicationPath),
+		Description: aws.String(fmt.Sprintf("ECS-CLI Application %s", app.Name)),
+		Type:        aws.String(ssm.ParameterTypeString),
+		Value:       aws.String(data),
+	})
+
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			switch aerr.Code() {
+			case ssm.ErrCodeParameterAlreadyExists:
+				return &store.ErrApplicationAlreadyExists{
+					ApplicationName: app.Name,
+					ProjectName:     app.Project}
+			}
+		}
+		return err
+	}
+
+	log.Printf("Created Application with version %v", *paramOutput.Version)
```

Comment:
What do you think about the store package **not** logging anything? I think that should be the responsibility of the `cli` package

Reply:
I don't mind removing them - but I was thinking this could be the verbose logging. We still have the task to add a verbose flag. #61

Reply:
sgtm, let's keep it!